ckanext/short_urls/model.py

```diff
@@ -0,0 +1,50 @@
+import string
+import random
+from sqlalchemy import Column, types, UniqueConstraint
+from sqlalchemy.ext.declarative import declarative_base
+from ckan.model.meta import metadata
+
+Base = declarative_base(metadata=metadata)
+
+OBJECT_TYPE_DATASET = 'dataset'
+OBJECT_TYPE_RESOURCE = 'resource'
```

Comment:
in python3 you can use Enum class for that https://docs.python.org/3/library/enum.html#using-automatic-values